docs/documentation/Upgrading.md

```diff
@@ -1,21 +1,36 @@
-### Upgrading
+## Upgrading
 
-## From 7.1.x to 8.0.x
+### From 8.0.x to 9.0.x
+
+* The type of [`datatable.data.data`](API#data) has changed from `object[]` to `{ cells: object[], attributes: object }[]`.
```

Comment:
Could we add a simple example of what this change means? Just a simple: Instead of X, do Y. So that people who don't read every commit message and just quickly want to upgrade from one to another version quickly figure out what to do.

Reply:
Updated with examples that showcase the changes and the new functionality to customize the row attributes.